news/2021-01-14-performance-points-updates.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Performance Points Updates"
+date: 2021-01-14 10:00:00 +0000
+---
+
+Thanks to the effort of a number of public contributors, we're excited to announce a new set of changes to osu!'s performance points (pp) algorithm that should help freshen things up at high levels of play. Read on for more details!
+
+![](/wiki/shared/news/2021-01-14-performance-points-updates/star-rating.jpg)
+
+Set off during the last few days of 2020, these new incoming changes represent an effort from various members of the community to try and address long-standing issues with one of the game's most important progression systems.
+
+For those unfamiliar with the technical minutae of the system, please consult [the wiki article](/wiki/Performance_points/) for an overview of the general ideas involved. 
+
+To summarize the article very briefly (you should really read the whole article), pp encompasses several core values as follows:
+
+- **Aim**, which considers how difficult it is to consistently hit notes back-to-back during play (affected by things like approach rate, jumps, Flashlight, Hidden, Hard Rock and more)
+- **Speed**, which considers how quickly new elements are presented to the player during the course of the map (affected by things like the number of hit objects over time, BPM, Double Time and Half Time)
+- **Accuracy**, which considers the player's ability to stay on time rhythmically and how difficult this is to achieve (affected by things like Hidden, Hard Rock, Flashlight and more)
+- **Strain**, which considers how often and how long a player has to endure intense or technically demanding patterns during play (affected by things like streams, fast jumps and more)
+
+**The changes listed below apply solely to the osu! game mode.**
+
+Let's get started with the details:
+
+## Approach Rate 11 Changes 
```

Comment:
I am fine with either approach. I did intentionally buff low AR as I felt the 1% buff was basically negligable, however I completely forgot about it during the MR. I dont believe that the change is detrimental, but since the change was not brought up in the original MR, it can totally be removed. Values from the change are still included in all changes, and very few plays / players benefit from them. As low AR is typically avoided for farming, I am confident that leaving the buff in will have a negligable effect. However, if mappers find a way to abuse this new low AR buff, it can be scaled back or adjusted or given a new formula entirely.

Reply:
After some voiced concerns I will be submitting a revert of the low AR buff later today.

Reply:
Lucky someone is paying attention, nice catch.